mxcubecore/HardwareObjects/LNLS/EPICS/EPICSActuator.py

```diff
@@ -0,0 +1,70 @@
+import threading
+import time
+
+import gevent
+import numpy as np
+
+from mxcubecore.HardwareObjects.abstract.AbstractActuator import AbstractActuator
+
+
+class EPICSActuator(AbstractActuator):
+    ACTUATOR_VAL = "val"  # setpoint
+    ACTUATOR_RBV = "rbv"  # readback
+
+    def __init__(self, name):
+        super(EPICSActuator, self).__init__(name)
+        self.setpoint = None
+        self._nominal_limits = (-1e4, 1e4)
+        self.default_timeout = 180
+        if not self.unit:
+            self.unit = 10**-3
+
+    def init(self):
```

Comment:
```suggestion
    def init(self):
        super().init()
```
Is it for purpose you are initialising the parent class?